Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -573,6 +578,103 @@ namespace AzToolsFramework
                 QFile::copy(oldPath.c_str(), newPath.c_str());
             }
         }
+
+        void AssetBrowserTreeView::MoveEntries()
+        {
+            using namespace AzFramework::AssetSystem;
+            EntryTypeFilter* foldersFilter = new EntryTypeFilter();
+            foldersFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Folder);
+
+            auto selection = AzToolsFramework::AssetBrowser::AssetSelectionModel::EverythingSelection();
+            selection.SetTitle("Pick folder to move to");
+            selection.SetMultiselect(false);
+            selection.SetDisplayFilter(FilterConstType(foldersFilter));
+            AssetBrowserTreeViewDialog dialog(selection, this);
+            dialog.exec();
+
+            const AZStd::vector<AZStd::string> folderPaths = selection.GetSelectedFilePaths();
+
+            if (!folderPaths.empty())
+            {
+                AZStd::string folderPath = folderPaths[0];
+                AZ_TracePrintf("JJS", "FolderPath = %s\n", folderPath.c_str());
+                bool connectedToAssetProcessor = false;
+                AzFramework::AssetSystemRequestBus::BroadcastResult(
+                    connectedToAssetProcessor, &AzFramework::AssetSystemRequestBus::Events::AssetProcessorIsReady);
+
+                if (connectedToAssetProcessor)
+                {
+                    auto entries = GetSelectedAssets();
+
+                    for (auto entry : entries)
+                    {
+                        using namespace AZ::IO;
+                        AZStd::string originalFname;
+                        AssetBrowserEntry* item = entry;
+                        Path fromPath = item->GetFullPath();
+                        PathView filename = fromPath.Filename();
+                        Path toPath(folderPath);
+                        toPath /= filename;
+                        AssetChangeReportRequest request(
+                            AZ::OSString(fromPath.c_str()), AZ::OSString(toPath.c_str()), AssetChangeReportRequest::ChangeType::CheckMove);
+                        AssetChangeReportResponse response;
+
+                        if (SendRequest(request, response))
+                        {
+                            AZStd::string message;
+                            for (int i = 0; i < response.m_lines.size(); ++i)
+                            {
+                                message += response.m_lines[i] + "\n";
+                            }
+
+                            if (message.size())
+                            {
+                                QMessageBox msgBox(this);
+                                msgBox.setWindowTitle("Pre-Move Report");
```

Comment:
Is this the title that users will see? If so, it could potentially be more user-friendly like "Move Asset" or whatever fits best. If "Pre-Move Report" is something users might expect to see and understand then ignore this.

Reply:
Changed to "Before Move Asset Information".